hardhat.config.ts

```diff
@@ -9,6 +9,7 @@ import "@typechain/hardhat";
 import "hardhat-deploy";
 import "hardhat-gas-reporter";
 import "solidity-coverage";
+import "@tenderly/hardhat-tenderly";
```

Comment:
I think we can remove this from here now that we discovered that tenderly needs to use the js hardhat config that we use to spin up the hardhat node

Reply:
done

Reply:
yeah i think you're right, linters etc are complaining without it